9/changelog.yaml

```
@@ -24,3 +24,9 @@ releases:
           Users can still install this collection with ``ansible-galaxy collection install community.skydive``."
         - The deprecated servicenow.servicenow collection has been removed from Ansible 7, but accidentally re-added to Ansible 8. It has been removed again from Ansible 9
           (https://github.com/ansible-community/community-topics/issues/246).
+      deprecated_features:
+        - The collection ``t_systems_mms.icinga_director`` has been renamed to ``telekom_mms.icinga_director``.
+            For now both collections are included in Ansible. The content in ``t_systems_mms.icinga_director``
+            will be replaced with deprecated redirects to the new collection in Ansible 9.0.0,
+            and these redirects will eventually be removed from Ansible. Please update your
```

Let's fix a version when this will happen:
```suggestion
            and these redirects will be removed from Ansible 11. Please update your
```